### PR TITLE
refactor(oauth): remove oauthdb.revokeAuthorizedClients

### PR DESCRIPTION
**Pull request by dannycoates**
## Because

- part of #5167

## This pull request

- pulls out the `oauthdb.revokeAuthorizedClients` thread
- removes some dead code

### CODE REVIEW

File: packages/fxa-auth-server/lib/oauthdb/index.js [hunk 1]
```diff
@@ -118,12 +118,5 @@ module.exports = (log, config) => {
         payload: oauthParams,
       });
     },
-
-    async revokeAuthorizedClient(sessionToken, oauthParams) {
-      oauthParams.assertion = await makeAssertionJWT(config, sessionToken);
-      return callRoute('/authorized-clients/destroy', {
-        payload: oauthParams,
-      });
-    },
```
Comment (dannycoates) [inline packages/fxa-auth-server/lib/oauthdb/index.js:127]: The point

File: packages/fxa-auth-server/lib/routes/oauth/proxied.js [hunk 2]
```diff
@@ -80,43 +78,6 @@ module.exports = (log, config, oauthService, db, mailer, devices) => {
         return oauthService.getScopedKeyData(sessionToken, request.payload);
       },
     },
-    {
-      method: 'POST',
-      path: '/oauth/id-token-verify',
```
Comment (dannycoates) [inline packages/fxa-auth-server/lib/routes/oauth/proxied.js:85]: This route will outlive `proxied.js` so I moved it to its own file

File: packages/fxa-auth-server/lib/server.js [hunk 3]
```diff
@@ -366,17 +356,6 @@ async function create(
   );
   server.auth.strategy('subscriptionsSecret', 'subscriptionsSecret');
 
-  server.auth.scheme(
-    'fxa-oauthServerJWT',
```
Comment (dannycoates) [inline packages/fxa-auth-server/lib/server.js:370]: This was used by an oauth->auth request for the profile, which got replaced at some point last year with a direct db call

**Review by vbudhram — COMMENTED** [review]
:shipit:

**Review by vbudhram — APPROVED** [review]

**Pull request merged**